sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/OpenAIClientTestBase.java

```diff
@@ -320,4 +361,56 @@ static void assertEmptyContentFilterResults(ContentFilterResults contentFilterRe
         assertNull(contentFilterResults.getViolence());
         assertNull(contentFilterResults.getSelfHarm());
     }
+
+    static void assertChatCompletionWednesday(ChatCompletions chatCompletions) {
```

Comment:
We can move to the final naming on these? (whatever that is 😆) 

*YourData?

Reply:
that's a good point. I think we could use BYOD as in (bring your own data) ?

Reply:
The spellchecker is going to get angry at me if I do that 🤔 I changed it to "CognitiveSearch" since that's the extension class name anyway.